WWTMVC5/WWTWeb/MarsMoc.aspx

```diff
@@ -22,7 +23,7 @@
     Bitmap bmp1 = null;
     Bitmap bmp2 = null;
 
-    if (level > 15)
+    if (level > 14)
```

Comment:
We should make sure to confirm that the maximum tile level in this plate file is in fact 14.

Reply:
How do we check that? This is the code running on the VM right now, so I figured it had the appropriate levels.

Reply:
The most convenient thing that I can think if is to run the code with the level 15 check, then try to request a level 15 tile and see if it works.

Really these kinds of limits shouldn't be hardcoded into the ASPX, but should instead come from the actual data file ... probably the existing code crashes if you go too deep, or something.

Reply:
Added a comment to that effect to look into that